src/main/routes/routes.ts

```diff
@@ -106,6 +106,7 @@ export default function(app: Application): void {
   app.get('/summary-of-publications', app.locals.container.cradle.summaryOfPublicationsController.get);
   app.get('/file-publication', app.locals.container.cradle.flatFileController.get);
   app.get('/sjp-public-list', app.locals.container.cradle.sjpPublicListController.get);
+  app.get('/accessibility-statement', app.locals.container.cradle.accessibilityStatementController.get);
```

Comment:
Very small change :) Can you alphabetically arrange new route? It should come after app.get('/', app.locals.container.cradle.homeController.get);

Reply:
Done :)